cli.js

```diff
@@ -80,7 +80,7 @@ var cli = meow([
 	'  ava --init foo.js',
 	'',
 	'Default patterns when no arguments:',
-	'test.js test-*.js test/**/*.js'
+	'test.js test-*.js test/**/*.js __tests__/**/*.js **/*.test.js'
```

Comment:
Oh, is my glob wrong? Should it be `**/__tests__/**`? That's what I did for my PR to `nyc`. Forgive my ignorance with `glob` :sweat_smile: